src/main/java/core/basesyntax/generator/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.generator;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
+        for (Map.Entry<String, Integer> entry : Storage.fruits.entrySet()) {
+            report.append(entry.getKey()).append(",").append(entry.getValue()).append("\n");
+        }
+        return report.toString();
```

Comment:
If the storage is empty, the method will return an empty string. Consider returning a message indicating that there are no records to report, if applicable.